extide/gradle/src/org/netbeans/modules/gradle/NbGradleProjectImpl.java

```diff
@@ -220,7 +282,9 @@ void detachAllUpdater() {
     }
 
     void dumpProject() {
```

Comment:
dump!? clear? release?

Reply:
`dumpProject` name was not changed in this PR.